project/Dependencies.scala

```diff
@@ -67,8 +67,7 @@ object Dependencies {
   }
 
   object Plugins {
-    // This should follow the version of sbt-protoc ScalaPB is depending on
```

Comment:
this was not honored, and there is nothing that really requires it anyway, so I removed this note